public/stylesheets/site/2.0/16-zone-system.css

```diff
@@ -219,7 +219,7 @@ SUBSECTIONS:
   width: 48%;
 }
 
-.admin_sessions-new form {
+.sessions-new .new-admin {
```

Comment:
That should be an underscore for `.new_admin`, sorry! I'll try to remember to leave a second comment rather than an edit next time.